src/milind/Exam_1.java

```diff
@@ -0,0 +1,46 @@
+package milind;
+
+import java.util.Scanner;
+
+public class Exam_1 {
+	
+	String maxLengthName(String[] arr) {
+		int maxLengthOfname = arr[0].length();
+		String maxLengthName = arr[0];
+		for(int index = 0; index < arr.length; index++) {
+			if(maxLengthOfname < arr[index].length()) {
+				maxLengthOfname = arr[index].length();
+				maxLengthName = arr[index];
+			}
+		}
+		System.out.println("Maximum length name is : " + maxLengthName + " total length is " + maxLengthOfname);
+		return maxLengthName;
+	}
+	
+	void displayEvencharFromName(String name) {
+		int nameLength = name.length();

```

Comment:
no need to create this extra variable. directly use name.length() whereever required.